pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/adminoperation/ParticipantDataFixupService.java

```diff
@@ -3,35 +3,63 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Comparator;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import com.google.gson.Gson;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.StringUtils;
 import org.broadinstitute.dsm.db.DDPInstance;
 import org.broadinstitute.dsm.db.dao.ddp.participant.ParticipantDataDao;
 import org.broadinstitute.dsm.db.dto.ddp.participant.ParticipantData;
 import org.broadinstitute.dsm.exception.DSMBadRequestException;
 import org.broadinstitute.dsm.exception.DsmInternalError;
-import org.broadinstitute.dsm.model.defaultvalues.ATDefaultValues;
-import org.broadinstitute.dsm.pubsub.WorkflowStatusUpdate;
-import org.broadinstitute.dsm.service.admin.AdminOperation;
 import org.broadinstitute.dsm.service.admin.AdminOperationRecord;
+import org.broadinstitute.dsm.service.elastic.ElasticSearchService;
+import org.broadinstitute.dsm.service.participantdata.ATParticipantDataService;
+import org.broadinstitute.dsm.service.participantdata.DuplicateParticipantData;
+import org.broadinstitute.dsm.service.participantdata.ParticipantDataService;
+import org.broadinstitute.dsm.util.ParticipantUtil;
+import org.broadinstitute.dsm.util.proxy.jackson.ObjectMapperSingleton;
 
 /**
- * AdminOperation service to asynchronously fix various ParticipantData issues
+ * AdminOperation service to asynchronously fix various ParticipantData issues.
+ * </p>
+ * The operation requires a fixupType attribute. The supported fixup types are:
+ * - atcpGenomicId: remove duplicate genomic IDs and participant exit statuses
```

Comment:
Since these strings are in `FixupType`, maybe move these comments to the enum and then reference the enum here in the class javadoc.

Reply:
Very nice!  Thank you.